src/sortStudents.ts

```diff
@@ -1,16 +1,53 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
+type SortOrder = 'asc' | 'desc';
+
+const calcAverageGrade = (grades: number[]):number => {
+  return grades
+    .reduce((a: number, b: number) => a + b) / grades.length;
+};
+
 // create SortOrder type
-export type SortOrder;
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  return [...students].sort((a, b) => {
+    switch (sortBy) {
```

Comment:
```suggestion
  return [...students].sort((a, b) => {
    let firstEl = a;
    let secondEl = b;
    
    if (order === 'decs') {
      firstEl = b;
      secondEl = a;
    }
    
    switch (sortBy) {
```
И тогда нигде больше не надо будет проверять ордер, всегда можно будет делать `firstEl[sortBy].localeCompare(secondEl[sortBy]);`